packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkEncodingEndToEnd.spec.ts

```diff
@@ -374,5 +384,52 @@ describe("End to end chunked encoding", () => {
 			const jsonableTree = mapCursorField(chunk.cursor(), jsonableTreeFromCursor);
 			assert.deepEqual([initialTree], jsonableTree);
 		});
+
+		it("Initializing tree creates uniform chunks with encoded identifiers", async () => {
+			const factory = new SharedTreeFactory({
+				jsonValidator: typeboxValidator,
+				forest: ForestType.Optimized,
+			});
+			const provider = new TestTreeProviderLite(1, factory, true);
+
+			// Workaround for getting the same id values from the TestTreeProviderLite's deterministic id compressor
```

Comment:
Is not clear if this is workaround is avoiding getting the same ids, or exists so that the same ids are gotten. Some rephrasing or additional detail would be good. 

It would be good to indicate if the value of this seed actually matters (ex: any value except 0 works, or if 0 is fine, just use 0 for clarity, or does it have to be exactly `0xdeadbeef` for some reason)_.

Reply:
I think I realize what you are doing here now: predicting what the defaulted id will end up being ahead of time.

Since id defaulting has its own tests elsewhere, I think a cleaner way to just use testIdCompressor, and explicitly provide the id when initializing: via `view.initialize({identifier: stableId});`

Reply:
Updated test to use the testIdCompressor, instead of using two separate id compressors with the same seed